docs/sources/mimir/manage/run-production-environment/production-tips/index.md

```diff
@@ -121,6 +121,21 @@ For these reasons, run the store-gateways on disks such as SSDs that have fast d
 The compactor requires a lot of disk space to download source blocks from the long-term storage and temporarily store the compacted block before uploading it to the storage.
 For more information about required disk space, refer to [Compactor disk utilization](../../../references/architecture/components/compactor/#compactor-disk-utilization).
 
+### Manage capacity for large tenants
+
+When working with large tenants, there are two compactor-specific settings to consider when planning or adjusting capacity:
+
+- `-compactor.split-groups`
+- `-compactor.split-and-merge-shards`
+
+For more information about how they work, refer to [Compaction algorithm]({{< relref "../architecture/components/compactor/index.md#compaction-algorithm" >}}).
+
+As a best practice, use 1 shard per every 8 million active series in a tenant.
+
+For example, for a tenant with 100 million active series, use approximately 12 shards. Use an even number for the shard count.
```

Comment:
The runbook mentions that the number should be a power of 2:

> Set -compactor.split-groups and -compactor.split-and-merge-shards to a value that is 1 for every 8M active series you have - rounded to the closest even number. So, if you have 100M series - 100/8 = 12.5 = value of 12. If you are using query sharding on the query frontend, it is recommended to use the next power of 2 instead to avoid extra work on the read path (a value of 16 instead of 12 in the example), [see this blog post for more info](https://grafana.com/blog/2022/04/19/how-grafana-mimirs-split-and-merge-compactor-enables-scaling-metrics-to-1-billion-active-series/).

[link](https://github.com/grafana/mimir/blob/60d58fded280964bac8d32ca23ee3deb829649fb/docs/sources/mimir/manage/mimir-runbooks/_index.md)